### PR TITLE
AvatarState & remove Chi for avatar & an Method

### PR DESCRIPTION
**Pull request by runefist**
- Fix final blow, but still needs work in the core, since AvatarState
can't always be used.
- Remove chi from Avatar, Avatar's are benders... Not fighters...
- Simple method to check if an avatar has been choosen

### CODE REVIEW

File: src/com/projectkorra/rpg/RPGMethods.java [hunk 1]
```diff
@@ -42,14 +44,14 @@ public static boolean checkEveryInterval(World world) {
 	}
 	
 	public static void cycleAvatar(BendingPlayer bPlayer) {
-		if (Bukkit.getOnlinePlayers().size() <= 1) return; //Don't bother with 1 person on...
```
Comment (OmniCypher-) [inline src/com/projectkorra/rpg/RPGMethods.java]: This comment was correct seeing as though the check was less than or equal to one.

Comment (runefist) [inline src/com/projectkorra/rpg/RPGMethods.java]: Yes? But the comment that was on it was wrong, I have not changed the code of it.
It said don't bother with 0 players online, but the actually comment should be don't both with 1 or less persons online, like the code is saying.

Comment (runefist) [inline src/com/projectkorra/rpg/RPGMethods.java]: I've changed it btw, it is correct now

**Comment by runefist** [comment]
Any idea when this is getting pulled in? If something is wrong with it, please tell me :)

**Comment by OmniCypher-** [comment]
Sorry this took a little while rune, I have been quite busy with school. Thanks for the contribution!

**Pull request merged**